src/libs/data-utils.js

```diff
@@ -24,9 +24,9 @@ import validate from 'validate.js'
 
 
 const warningBoxStyle = {
-  backgroundColor: colors.warning(1),
+  backgroundColor: colors.warning(0.15),
   padding: '1rem 1.25rem',
-  color: colors.light(1), fontWeight: 'bold', fontSize: 12
+  color: colors.dark(), fontWeight: 'bold', fontSize: 12
```

Comment:
This is being overridden by a style in line at 366, so it's still showing up as white-ish

Reply:
Also line 368 the icon also needs the dark color rather than the light

Reply:
Is this new change (see below) what you meant should be done?

Reply:
I'm testing based on the mocks in the ticket, this first photo is what I see when I deploy your branch:
![image](https://user-images.githubusercontent.com/54322292/76015874-df010b80-5ee9-11ea-8514-0d2fc5fec27e.png)

and this photo is what the mocks look like:
![image](https://user-images.githubusercontent.com/54322292/76015960-0657d880-5eea-11ea-8bf2-c3a1c1f57b77.png)

so the text color seems correct now but the icon color still doesn't match.

Reply:
ah okay! I resolved that now I believe.